engine/src/main/java/org/terasology/math/ChunkMath.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2013 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.terasology.math;
+
+import org.terasology.math.geom.Vector3f;
+import org.terasology.world.chunks.ChunkConstants;
+
+/**
+ * Collection of math functions.
+ *
+ * @author Benjamin Glatzel <benjamin.glatzel@me.com>
```

Comment:
IMHO as long as we remember to update them for new/cloned files it doesn't really hurt to have them. It can give a clue to who might relate to something, and it can be nice sometimes to see your name in an author tag. Matters more to some than others, but doesn't really hurt.

But you are correct that it isn't necessarily very relevant and isn't very needed against something more formal like Git history :-)

Dunno if we'd want to perhaps try making them more useful, by having some better guidelines for when to add yourself to a file.